### PR TITLE
Time synch note

### PR DESCRIPTION
**Pull request by Feediver1**
## Description

Resolves https://redpandadata.atlassian.net/browse/DOC-20
Review deadline: Weds Nov 27

## Page previews

[Preview](https://deploy-preview-875--redpanda-docs-preview.netlify.app/current/develop/produce-data/configure-producers/#broker-timestamps)

## Checks

- [ ] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

**Comment by Feediver1** [comment]
@0xdiba This is an old ticket--not sure if still applicable. Please advise.

**Review by 0xdiba — APPROVED** [review]
@Feediver1 yes that's a very old discussion.
The new doc looks good to me 👍

File: modules/develop/pages/produce-data/configure-producers.adoc [hunk 1]
```diff
@@ -217,6 +217,8 @@ in one place, while also enabling features like schema evolution.
 
 Redpanda employs a unique strategy to help ensure the accuracy of retention operations. In this strategy, closed segments are only eligible for deletion when the age of all messages in the segment exceeds a configured threshold. However, when a producer sends a message to a topic, the timestamp set by the producer may not accurately reflect the time the message reaches the broker. To address this time skew, each time a producer sends a message to a topic, Redpanda records the broker's system date and time in the `broker_timestamp` property of the message. This property helps maintain accurate retention policies, even when the message's creation timestamp deviates from the broker's time.
 
+NOTE: Clock synchronization should be monitored by the server owner, as Redpanda does not monitor clock synchronization. While Redpanda does not rely on clocks for correctness, if  you are using LogAppendTime, server clocks may affect the time your application sees.
```
Comment (Deflaimun) [inline modules/develop/pages/produce-data/configure-producers.adoc]:
```suggestion
NOTE: Clock synchronization should be monitored by the server owner, as Redpanda does not monitor clock synchronization. While Redpanda does not rely on clocks for correctness, if you are using `LogAppendTime` (server timestamp set by Redpanda), server clocks may affect the time your application sees.
```

Comment (Deflaimun) [reply modules/develop/pages/produce-data/configure-producers.adoc]: I see here that we talk about what is LogAppendTime https://deploy-preview-875--redpanda-docs-preview.netlify.app/current/develop/produce-data/configure-producers/#disable-broker-timestamp-retention

Should we move this parenthesis to this new note? 

As a new user LogAppendTime doesn't mean much. But looks like it's a common Kafka thing https://stackoverflow.com/questions/74286923/kafka-streams-create-time-vs-log-append-time

Comment (Feediver1) [reply modules/develop/pages/produce-data/configure-producers.adoc]: Thanks @Deflaimun Added the parenthetical here, but think it doesn't hurt to keep in the linked topic as well.

**Review by Deflaimun — APPROVED** [review]

**Pull request merged**